### PR TITLE
Add "Getting started with rust-hwi" blog

### PR DESCRIPTION
**Pull request by wszdexdrf**
A blog post about the basic fundamentals of rust-hwi and integration with BDK.

### CODE REVIEW

File: docs/_blog/getting_started_with_rust_hwi.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+title: "Getting Started with rust-hwi"
+description: "This post will help one understand and develop for hardware wallets using BDK"
+authors:
+    - Wszdexdrf
+date: "2022-08-16"
+tags: ["BDK", "Development", "Hardware Wallets"]
+hidden: true
+draft: false
+---
+# Getting started with rust-hwi
```
Comment (csralvall) [inline docs/_blog/getting_started_with_rust_hwi.md]: It seems that this title isn't necessary as the deployment takes the title from the metatags above.
You can see they are duplicated in the blog preview.

Comment (wszdexdrf) [reply docs/_blog/getting_started_with_rust_hwi.md]: Thanks for pointing that out.

File: docs/_blog/getting_started_with_rust_hwi.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+Then we store the first device available into `device`, which is straightforward. We then call `HWIClient::get_client()` and pass the reference to the device info and the Chain we are going to use that device on. (the boolean is for setting `expert` mode, which allows for some more functions and detailed information. That is implemented in bitcoin-core/HWI, see the [docs](https://hwi.readthedocs.io/en/latest/usage/cli-usage.html#cmdoption-hwi-expert)) There are 4 chains available, as usual, Main, Test, Regtest & Signet.
+
+HWI contains a base class known as `HardwareWalletClient`. All the functions and their arguments are defined in it. Hardware Wallet developers create their own implementations of the base class. The function `get_client()` returns an instance of `HWIClient` struct, which contains a reference to the Python object of `HardwareWalletClient` (and also a reference to the Python code of HWI itself). rust-hwi's `HWIClient` tries to mimic the base class `HardwareWalletClient` and thus all the functions used for communicating with a hardware wallet belong to `HWIClient`.
```
Comment (notmandatory) [inline docs/_blog/getting_started_with_rust_hwi.md]: Some minor suggestions to clarify what is a python class and what is in rust.
```suggestion
HWI contains a python base class known as `HardwareWalletClient`. All the functions and their arguments are defined in it. Hardware Wallet developers create their own implementations of the base class. The function `get_client()` returns an instance of `HWIClient` struct, which contains a reference to the Python object of `HardwareWalletClient` (and also a reference to the Python code of HWI itself). rust-hwi's `HWIClient` struct tries to mimic the base class `HardwareWalletClient` and thus all the functions used for communicating with a hardware wallet belong to `HWIClient`.
```

Comment (wszdexdrf) [reply docs/_blog/getting_started_with_rust_hwi.md]: Thanks for pointing it out.

File: docs/_blog/getting_started_with_rust_hwi.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    MemoryDatabase::default(),
+)?;
+wallet.add_signer(
+    KeychainKind::External,
+    SignerOrdering(200),
+    Arc::new(custom_signer),
+);
+```
+
+Rest everything remains the same!
```
Comment (notmandatory) [inline docs/_blog/getting_started_with_rust_hwi.md]: Another small language nit for clarity.
```suggestion
The rest of the PSBT signing process remains the same!
```

**Comment by notmandatory** [comment]
Nice write up! clear and to the point. Should we put this in the "Tutorials" section?

**Comment by wszdexdrf** [comment]
> Nice write up! clear and to the point. Should we put this in the "Tutorials" section?

That would be fantastic! Do I move the file to the tutorials folder or it will be done in some other way?

**Comment by notmandatory** [comment]
> That would be fantastic! Do I move the file to the tutorials folder or it will be done in some other way?

Yes, just move it to the tutorial folder. Also the other tutorials begin with "Introduction" , so you should rename "About" to that.

**Review by notmandatory — APPROVED** [review]
ACK 4f7d14dc6f33011553aa9ed176483e7f276260fa

**Pull request merged**